web-registry/src/components/organisms/ProjectDetailsSection/ProjectDetailsSection.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, useMediaQuery, useTheme } from '@mui/material';
+
+import { BlockContent } from 'web-components/lib/components/block-content';
+import { CredibilityCard } from 'web-components/lib/components/cards/CredibilityCard/CredibilityCard';
+import Section from 'web-components/lib/components/section';
+import ResponsiveSlider from 'web-components/lib/components/sliders/ResponsiveSlider';
+import { Body, Label, Title } from 'web-components/lib/components/typography';
+import { Theme } from 'web-components/lib/theme/muiTheme';
+
+import { useSectionStyles } from './ProjectDetailsSection.styles';
+import { ProjectDetailsSectionProps } from './ProjectDetailsSection.types';
+
+export const ProjectDetailsSection: React.FC<ProjectDetailsSectionProps> = ({
+  header,
+  credibilityCards,
+}) => {
+  const theme = useTheme<Theme>();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
Same comment as above, would be nice to have that without client MQ check to be more SSR friendly.

Reply:
I understand, let's keep it like that then.